package.json

```diff
@@ -5,19 +5,20 @@
   "main": "index.js",
   "scripts": {
     "pretest": "npm install",
-    "test": "eslint lib/**/*.js test/**/*.js index.js && mocha --exit --require should --reporter spec --check-leaks"
+    "test": "eslint lib/**/*.js test/**/*.js index.js && mocha --exit --require should --reporter spec --check-leaks",
+    "test-http2": "HTTP2_TEST=1 mocha --exit --require should --reporter spec --check-leaks"
   },
   "dependencies": {
     "methods": "~1.1.2",
-    "superagent": "3.8.2"
+    "superagent": "git+https://github.com/visionmedia/superagent.git"
```

Comment:
Maybe point back to `superagent@^8.0.0` now that https://github.com/visionmedia/superagent/pull/1414 has been merged.

PS: `superagent@^8.0.0` is in the [master branch package.json](https://github.com/visionmedia/supertest/blob/5543d674cf9aa4547927ba6010d31d9474950dec/package.json#L36) at the time of writing.